### ISSUE TITLE
[RFC] Support for logging batches of metrics/params/tags

### ISSUE THREAD

**Issue by smurching** [mlflow/mlflow#908]
## TL;DR:
[This google doc](https://docs.google.com/document/d/1hOymz5TJPdNwRD0O7Ne94dfsGjGeq5q8g8-8Src85M4/edit#) contains a proposal for adding new REST, fluent, and client APIs to MLflow to enable logging batches of metrics, params, and tags. The primary goal here is to improve performance by allowing logging of multiple entities in a single REST API request.

We'd like to introduce the new APIs in MLflow 1.0 & potentially deprecate the existing LogMetric, LogParam, SetTag REST APIs (the `log_metric` etc fluent & client APIs would be preserved). Any feedback (via google doc or GitHub issue comments) is welcome!

**Comment by dbczumar** [comment]
Closing this RFC because the described features have been implemented.
